### ISSUE TITLE
Configuration files for database applications

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-robotlog2db#29]
Currently we have two database applications (RobotLog2DB and PyTestLog2DB). Every application is prepared to handle the XML result files of a certain test framework (Robot Framework and pytest). The XML files are different, but the basic functionality is the same in both database applications. Both applications provide a mapping mechanism between a test case and a component name. Because of different XML files also the mapping is different:

Class names in pytest files, e.g:
`"Testsuite1" "test-data.test_tsclass.TestSuite1",`

Paths in Robot Framework files, e.g:
`"cli" : "robot/cli",`

What is the impact? Does this mean that also the configuration files are specific for a certain database application?
Does this mean that we need to maintain two separate configuration files, even in case of remaining values like versions
are the same? Do we have any use case that requires this file separation? In my opinion the file separation is unfavorable.

Can the parsing mechanism of the database applications be used for both configuration types?

I would prefer to have to maintain only one configuration file containing the configuration for all together.

Maybe in this way:

```
"my_component" [
                  "test-data.test_mycomponentclass.TestMyComponent",
                  "robot/my_component/testresults"
               ]
```

Or in this way:

```
"my_component" [
                  ["pytest", "test-data.test_mycomponentclass.TestMyComponent"],
                  ["robotframework", "robot/my_component/testresults"]
               ]
```
And every database application picks out of the configuration file what belongs to her.

**Comment by HolQue** [comment]
In between I tested this. Common format works. Extensions mentioned above, are not required.

Should be mentioned in documentation (of both database apps) that the configuration format is compatible and therefore both apps can use a common config file.

**Comment by HolQue** [comment]
But caution: The filter strings must be really unique in case of they are merged within a common config file!

**Comment by test-fullautomation** [comment]
Hi Ngoan,

please take care that the documentation is the same for all \*log2db\* tools.

Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
solved with #44
